src/ucp/core/ucp_context.c

```diff
@@ -223,6 +223,7 @@ static ucs_config_field_t ucp_context_config_table[] = {
    "Communication scheme in RNDV protocol.\n"
    " get_zcopy - use get_zcopy scheme in RNDV protocol.\n"
    " put_zcopy - use put_zcopy scheme in RNDV protocol.\n"
+   " am        - use am scheme in RNDV protocol.\n"
```

Comment:
Maybe remove that line as #9154 is already taking of this?

Reply:
Done, tnx!